orchagent/intfsorch.cpp

```diff
@@ -416,6 +430,62 @@ bool IntfsOrch::setIntfProxyArp(const string &alias, const string &proxy_arp)
     return true;
 }
 
+bool IntfsOrch::setIntfLoopbackAction(const Port &port)
```

Comment:
if you pass loopback action as a string to this function, no need for `getIntfLoopbackActionStr`

Reply:
Done.